src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -10,11 +10,11 @@
 
 #include <DefaultSettings.h>
 #include <unicode.hpp>
+#include <utils.hpp>

```

Comment:
The issue was reported here: https://github.com/microsoft/terminal/issues/9232#issuecomment-1777044998

This PR changes a lot of whitespace indentation, so use this: https://github.com/microsoft/terminal/pull/16224/files?w=1